src/fsharp/PrettyNaming.fs

```diff
@@ -131,10 +131,20 @@ module internal Microsoft.FSharp.Compiler.PrettyNaming
         let nameLen = name.Length
         let rec loop i = (i < nameLen && (opCharSet.Contains(name.[i]) || loop (i+1)))
         loop 0
+    
+    /// Returns the plain name of a given identifier, operator or double backticked name, 
+    /// e.g. "|>>" instead of "( |>> )" or "long identifier" instead of "( long identifier )".
+    let DemangleBacktickedName (name:string) =
+        if name.StartsWith "( " && name.EndsWith " )" then
+            name.[2..name.Length - 3]
+        else name
+    
+    /// Wraps double backticks around a name if it contains a space.
+    let NameUsedInCode (name:string) = if name.Contains " " then "``" + name + "``" else name
```

Comment:
The logic for using double backticks is more complicated than this. Take a look at QuoteIdentifierIfNeeded in `src/fsharp/lexhelp.fs` - ideally use that function instead